client/src/helpers/ehr-text.js

```diff
@@ -1,17 +1,19 @@
 export const Text = {
   // STUDENT_INTRO: 'Click the edit icon to fill in the form.'
-  STUDENT_HAS_SUBMITTED: 'This assignment has been sent to your instructor for evaluation. It is no longer editable.',
```

Comment:
Why are you getting rid of this message? I don't see anything to replace it here. We need to be able to tell the student once they've submitted, that, well, they've submitted. :)